repositories/user.repository.js

```diff
@@ -0,0 +1,31 @@
+var mongoose = require('mongoose');
+
+var UserSchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true
+    },
+    password: {
+        type: String,
+        required: true
+    },
+    firstname: {
+        type: String,
+        required: true
+    },
+    lastname: {
+        type: String,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true
+    },
+    contact: {
+        type: Number,
```

Comment:
Change this type to `String`. We often only use `Number` if we need to do arithmetic operations.

Reply:
done